miriway.desktop.in

```diff
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Name=Miriway
+Comment=Mir/Wayland desktop environment
+Exec=env MIRIWAY_SESSION_STARTUP=@CMAKE_INSTALL_FULL_LIBEXECDIR@/miriway-session-startup sh -l -c miriway
```

Comment:
This should probably be a shipped as a simple wrapper executable script so that it can also have the `TryExec` line set to it too.